lib/karafka/server.rb

```diff
@@ -20,6 +25,7 @@ def process
       def bind_on_sigint
         process.on_sigint do
           Karafka::App.stop!
```

Comment:
Also I'd like to see definition of `process` function under the `bind_on_sigint` and `bind_on_sigquit `. Because it is used inside of it.